packages/cactus-core-api/src/main/kotlin/generated/openapi/kotlin-client/src/main/kotlin/org/openapitools/client/models/LedgerType.kt

```diff
@@ -40,6 +40,9 @@ enum class LedgerType(val value: kotlin.String) {
     @Json(name = "CORDA_4X")
     cORDA4X("CORDA_4X"),
 
+    @Json(name = "ETHEREUM")
+    eTHEREUM("ETHEREUM"),
+
     @Json(name = "FABRIC_2")
```

Comment:
Please remove all sdks, including Kotlin. We will add them if needed

Reply:
this was generated automatically by ```yarn codegen```, I'm not sure what you mean by removing